strawberry_django/mutations/resolvers.py

```diff
@@ -67,67 +69,81 @@ def _parse_pk(
         return value.parse(model)
 
     if isinstance(value, dict):
+        if key_attr in value:
+            obj_pk = value[key_attr]
+            if obj_pk != strawberry.UNSET:
+                if key_attr == "id":
+                    obj_pk = int(obj_pk)
```

Comment:
I think this can lead to issues. e.g. if your `id` is a `UUID` or something else

Note that `SomeModel.objects.get(pk='1234')` will work fine as django itself will cast the string to int if the column is an int

```suggestion
```

Reply:
Not related to your PR :)

Thanks for this!